diff -Naur a/zorder-lib/Makefile b/zorder-lib/Makefile
--- a/zorder-lib/Makefile	2015-03-12 11:57:33.000000000 -0400
+++ b/zorder-lib/Makefile	2020-02-11 16:31:54.355544824 -0500
@@ -2,7 +2,7 @@
 # Version: $Name: Apr24-2015 $
 # $Revision: 1.3 $
 
-ZO_LIB=zorder-lib
+ZO_LIB=libzorder
 ZO_TEST_PROGS = zo-sanity1 zo-sanity2 zo-sanity3 zo-sanity4
 
 ALL:	$(ZO_LIB).a $(ZO_TEST_PROGS)
@@ -15,6 +15,8 @@
 
 ZO_LIB_OBJECTS = $(ZO_LIB_SOURCES:.c=.o)
 
+PREFIX ?= /usr/local
+
 CC = gcc
 OPT = -g
 CFLAGS = $(OPT) -I. -Wall
@@ -60,3 +62,8 @@
 # deps
 zorder.o:	zorder.c zorder.h
 
+install:	zorder.o $(ZO_LIB).a zorder.h
+	install -d $(DESTDIR)$(PREFIX)/lib/
+	install -m 644 zorder.o $(ZO_LIB).a $(DESTDIR)$(PREFIX)/lib/
+	install -d $(DESTDIR)$(PREFIX)/include/
+	install -m 644 zorder.h $(DESTDIR)$(PREFIX)/include/
diff -Naur a/zorder-lib/zorder.h b/zorder-lib/zorder.h
--- a/zorder-lib/zorder.h	2015-03-12 11:57:38.000000000 -0400
+++ b/zorder-lib/zorder.h	2020-02-11 16:31:54.355544824 -0500
@@ -77,6 +77,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <math.h>
+#include <string.h>
 
 
 /* status defines */
